cli/src/commands/new.rs

```diff
@@ -37,12 +42,20 @@ impl NewCmd {
         // Remove the .git directory.
         fs::remove_dir_all(root.join(".git"))?;
 
-        // Check if the user has `foundry` installed.
-        if Command::new("foundry").arg("--version").output().is_err() {
-            println!(
-                "    \x1b[1m{}\x1b[0m Make sure to install Foundry to use contracts: https://book.getfoundry.sh/getting-started/installation.",
+        if self.evm {
+            // Check if the user has `foundry` installed.
+            if Command::new("foundry").arg("--version").output().is_err() {
+                println!(
+                "    \x1b[1m{}\x1b[0m Make sure to install Foundry to use contracts: https://book.getfoundry.sh/getting-started/installation",
```

Comment:
removing period because this directs to [https://book.getfoundry.sh/getting-started/installation.](https://book.getfoundry.sh/getting-started/installation.) which fails